var/spack/repos/builtin/packages/launchmon/package.py

```diff
@@ -25,25 +25,46 @@
 from spack import *
 
 
-class Launchmon(Package):
+class Launchmon(AutotoolsPackage):
     """Software infrastructure that enables HPC run-time tools to
        co-locate tool daemons with a parallel job."""
     homepage = "https://github.com/LLNL/LaunchMON"
     url = "https://github.com/LLNL/LaunchMON/releases/download/v1.0.2/launchmon-v1.0.2.tar.gz"
 
+    version('1.0.3b', git='https://github.com/LLNL/LaunchMON.git',
+            commit='1587a59d18e1d5159e441220e5686eaaa649647c')
     version('1.0.2', '8d6ba77a0ec2eff2fde2c5cc8fa7ff7a')
-
     depends_on('autoconf', type='build')
     depends_on('automake', type='build')
     depends_on('libtool', type='build')
+    depends_on('pkg-config', type='build')
     depends_on('libgcrypt')
     depends_on('libgpg-error')
+    depends_on("libelf")
+    depends_on("boost")
+
+    # A patch to hack around need to boostrap the git commit version
+    patch('lmon103b.patch', when='@1.0.3b')
 
-    def install(self, spec, prefix):
-        configure(
-            "--prefix=" + prefix,
-            "--with-bootfabric=cobo",
-            "--with-rm=slurm")
+    def setup_environment(self, spack_env, run_env):
+        # These env var settings are temporary until the LaunchMON build
+        # system automatically detects the appropriate libraries
+        if self.spec.satisfies('arch=linux-redhat7-ppc64le'):
+            spack_env.set('LD_LIBRARY_PATH', '/opt/ibm/spectrum_mpi/lib')
+            spack_env.set('LDFLAGS',
+                          '-L/opt/ibm/spectrum_mpi/lib '
+                          ' -Wl,-rpath=/opt/ibm/spectrum_mpi/lib '
+                          '-lopen-rte -lopen-pal')
```

Comment:
I don't think we should be merging this with the hardcoded spectrum MPI paths.  Will things work if you add a dependency on 'mpi', then pointed it at the spectrum MPI?  I have an uncommited spectrum MPI package in the Spack tcetmp install on rzmanta.